pom.xml

```diff
@@ -365,6 +366,7 @@
                         </goals>
                         <configuration>
                             <suffixes>(?&lt;!IntegrationSuite)</suffixes>
+                            <skipTests>${skip.unit.tests}</skipTests>
```

Comment:
To understand, why was this one needed?

Reply:
So I can run parallel integration tests and unit tests

Reply:
Sorry for being slow, how does this enable that? Can you override the pom property in the yaml file of the action?

Reply:
Or is `true` the default?

Reply:
false is the default. False needs to be set using `-Dskip.unit.tests=true`. That is used in integration tests.

Reply:
Understood now. The property is overridden in the job in the file above.